### PR TITLE
docs: update docs

### PR DESCRIPTION
**Pull request by ra0x3**
### Description
- As you can see, the plugin docs were a bit dated, this PR changes that

### Testing steps
- [ ] N/A

### Changelog
- docs: bring plugin docs up to date

### CODE REVIEW

File: plugins/forc-index/src/commands/revert.rs [hunk 1]
```diff
@@ -4,22 +4,22 @@ use clap::Parser;
...
+    #[clap(short, long, help = "Path of indexer project.")]
     pub path: Option<PathBuf>,
 
-    /// Path to the manifest of indexer project being deployed.
+    /// Path to the manifest of the indexer project being reverted.
     #[clap(
         short,
         long,
-        help = "Path to the manifest of indexer project being deployed."
+        help = "Path to the manifest of indexer project being reverted."
```
Comment (deekerno) [inline plugins/forc-index/src/commands/revert.rs:22]: Should this match the docstring above?

File: plugins/forc-index/src/commands/remove.rs [hunk 2]
```diff
@@ -3,11 +3,11 @@ use anyhow::Result;
...
+/// Stop and remove a running indexer.
 #[derive(Debug, Parser)]
 pub struct Command {
-    /// URL at which index is deployed.
-    #[clap(long, default_value = defaults::INDEXER_SERVICE_HOST, help = "URL at which index is deployed.")]
+    /// URL at which indexer is deployed.
+    #[clap(long, default_value = defaults::INDEXER_SERVICE_HOST, help = "URL at which indexer is deployed.")]
     pub url: String,
 
     /// Path to the manifest of indexer project being deployed.
```
Comment (deekerno) [inline plugins/forc-index/src/commands/remove.rs:13]: Should this also be changed to

> Path to the manifest of _the_ indexer project being deployed

to mirror the change in the revert operation?

**Review by deekerno — CHANGES_REQUESTED** [review]
Looks good overall! Just some comments about consistency.

File: plugins/forc-index/src/commands/build.rs [hunk 3]
```diff
@@ -42,10 +42,10 @@ pub struct Command {
     #[clap(long, help = "Building for native execution.")]
     pub native: bool,
 
-    /// Path with which to prefix asset filepaths in the index manifest.
+    /// Path with which to prefix asset filepaths in the indexer manifest.
     #[clap(
```
Comment (rvmelkonian) [inline plugins/forc-index/src/commands/build.rs:46]: this comment / help has always confused me. Suggested change (or something similar) 
`/// output path of indexer assets in the manifest`

File: plugins/forc-index/src/commands/deploy.rs [hunk 4]
```diff
@@ -55,10 +54,10 @@ pub struct Command {
     #[clap(long, help = "Building for native execution.")]
     pub native: bool,
 
-    /// Path with which to prefix asset filepaths in the index manifest.
+    /// Path with which to prefix asset filepaths in the indexer manifest.
```
Comment (rvmelkonian) [inline plugins/forc-index/src/commands/deploy.rs:57]: If above comment is used, update this one as well.

File: plugins/forc-index/src/commands/init.rs [hunk 5]
```diff
@@ -6,29 +6,29 @@ use std::path::PathBuf;
...
         parse(from_os_str),
-        help = "Path at which to create index."
+        help = "Path at which to create indexer."
     )]
     pub path: Option<PathBuf>,
 
-    /// Namespace in which index belongs.
-    #[clap(long, help = "Namespace in which index belongs.")]
+    /// Namespace in which indexer belongs.
+    #[clap(long, help = "Namespace in which indexer belongs.")]
```
Comment (rvmelkonian) [inline plugins/forc-index/src/commands/init.rs:23]: In the file below `new.rs` we have `/// Name of the indexer namespace`. I'm neutral on which one we choose, but they should be the same for consistency.

**Review by rvmelkonian — CHANGES_REQUESTED** [review]
Thanks for updating this. The old manifest args in the docs caused some confusion for me also. Just left a few comments.

File: plugins/forc-index/src/commands/build.rs (hunk 3 above)
Comment (ra0x3) [reply plugins/forc-index/src/commands/build.rs:46]: @rvmelkonian I agree this is confusing. It's actually cleaned up in https://github.com/FuelLabs/fuel-indexer/pull/676 (so don't mind it here)

**Review by rvmelkonian — APPROVED** [review]

**Pull request merged**